src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -354,15 +370,15 @@ describe('LoanDetails', () => {
         return true;
       });
       const updatedPropsData = {
-        ...propsData,
+        ...claimedReturnedPropsData,
         loanAccountActions: [{}, {}],
       };
       renderLoanProxyDetails(updatedPropsData);
       userEvent.click(screen.queryAllByRole('button')[5]);
       userEvent.click(screen.queryAllByRole('button')[9]);
       expect(screen.getAllByText('ui-users.loans.markAsMissing')).toBeTruthy();
     });
-    it.skip('Fee Fine Else Condition More than 1 loanAccountActions with open item', () => {
+    it('Fee Fine Else Condition More than 1 loanAccountActions with open item', () => {
```

Comment:
Could you please improve the test title to better elucidate the aim of the unit test?